src/nvidia.cc

```diff
@@ -243,6 +244,9 @@ const int translate_nvidia_attribute[] = {
 
     NV_CTRL_GPU_CURRENT_PERFORMANCE_LEVEL,
     NV_CTRL_IMAGE_SETTINGS,
+
+    NV_CTRL_STRING_PRODUCT_NAME,
```

Comment:
Seems NV_CTRL_STRING_PRODUCT_NAME was implicitly translated before (to zero). Added it for clarity/readability of the lists.